sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StatusTests.cs

```diff
@@ -1,61 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Removing this file. I am not sure why this was added separately. Should validate this as part of individual telemetry types i.e. RequestData and RemodeDependencyData.